recipes/conversation_watsonxai/package.json

```diff
@@ -27,7 +27,13 @@
     "eslint-plugin-import": "^2.22.1"
   },
   "dependencies": {
-    "@ibm-generative-ai/node-sdk": "^1.4.2",
-    "tjbot": "file:../../../tjbotlib"
+    "@iarna/toml": "^2.2.5",
+    "@ibm-cloud/watsonx-ai": "^1.1.0",
+    "axios": "0.27.2",
+    "dotenv": "^16.4.5",
+    "https": "^1.0.0",
+    "import-meta-resolve": "^4.0.0",
+    "readline-sync": "^1.4.10",
+    "tjbot": "^2.0.1"
```

Comment:
Should be `^3.0.0`, the recipe won't work on earlier versions of `node-tjbotlib`.